alert-common/src/test/java/com/synopsys/integration/alert/common/rest/model/AlertPagedModelTest.java

```diff
@@ -0,0 +1,48 @@
+package com.synopsys.integration.alert.common.rest.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class AlertPagedModelTest {
+    int testTotalPages = 1;
+    int testCurrentPage = 2;
+    int testPageSize = 3;
+    List<AlertSerializableModel> testContent;
+
+    @BeforeEach
+    public void init() {
+        testContent = new ArrayList<>();
+        AlertSerializableModel testAlertSerializableModel = Mockito.mock(AlertSerializableModel.class);
+        testContent.add(testAlertSerializableModel);
+    }
+
+    @Test
+    public void getTotalPagesTest() {
+        AlertPagedModel<AlertSerializableModel> testAlertPagedModel = new AlertPagedModel<>(testTotalPages, testCurrentPage, testPageSize, testContent);
+        assertEquals(testTotalPages, testAlertPagedModel.getTotalPages());
+    }
```

Comment:
All of these methods look quite similar. You can define testAlertPagedModel in the @BeforeEach method instead of manually doing it every time.